### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by UtkarshJain98**
Updating the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,4 @@
 # Sisu Looker Insights
+To install or update the application, please contact your Looker admin to install it from `<YourLookerInstance>/marketplace/view/sisu-insights`
```
Comment (jwyung) [inline README.md:2]:
```suggestion
To install or update the application, please contact your Looker admin to install it from `<YourLookerInstance>/marketplace/view/sisu-insights`.
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,4 @@
...
->kal% git commit . -m 'bumped the deploy'
->
->kal% git push
->
-then go to github, merge the PR, get the new main sha.
-
-finally, go to the looker space and say, "Hey google team - we have a new sha we'd like to deploy on experiment-sisu-staging.dev.looker.com. It's BIGSCARYSHA. Thanks!
-
-enjoy a beverage of your choice as google chugs through it and deploys it.
+If you would like to install the latest changes from this repo, please contact support@sisudata.com
```
Comment (jwyung) [inline README.md:4]:
```suggestion
If you would like to install the latest changes from this repo, please contact support@sisudata.com.
```

**Review by jwyung — APPROVED** [review]
Thanks @UtkarshJain98 ! What about the sections about building for dev or building a release candidate? Do we need to remove those?

File: README.md (hunk 2 above)
Comment (UtkarshJain98) [reply README.md:4]: @jaisundararajan suggested we dont add a fullstop at the end of a link. 
Can update based on whatever the decision is

**Comment by UtkarshJain98** [comment]
> What about the sections about building for dev or building a release candidate? Do we need to remove those?

The suggestion from Jai was to only have a basic readme pointing to links for installing Sisu Insights. Any references on building and releasing should be made to internal docs that only engg can reference. I think the doc that @yingan-wang created (https://www.notion.so/sisudata/Looker-Test-and-Release-Script-2afbc9c0919247efa4ca86097dc3cc27) covers the steps for it.

**Review by jwyung — APPROVED** [review]
Thanks!

**Pull request merged**